[PL-Doc] CD - Datadog notification updates (#9134)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/cd-steps/notify-users-of-pipeline-events.md

</Tabs>

## Datadog notifications

> The Datadog Notifications in Harness allows you to send pipeline event notifications directly to your Datadog monitoring system using the Events API. 

Reviewer: Please add FF details as well.

Author: done.

This integration helps you keep track of important pipeline updates within your existing monitoring workflows.

### Prerequisites
  

>   - API Key for your Datadog account. 

Reviewer: Please add URL as well as a Pre-requisite above API key.

Author: datadog URL?

Reviewer: Yes

Author: i have added DD url.

    - You can generate one from the API Keys section under `Profile -> Organization Settings -> API Keys` in Datadog.

### Configuration

>   - Give name to your datadog notification

Reviewer: ```suggestion
  - Give a name to your Datadog notification channel.
```

  - Select Pipeline Events as per your requirements.

>   - Select Channel type as `Datadog(/v1/eventsAPI)` in Notification Method.

Reviewer: ```suggestion
  - Select Channel type as `Datadog(/v1/events API)` in Notification Method.
```

    ![datadog-selection](./static/datadog-notification-1.png)

>   - Provide the URL and API key after selecting `Datadog(v1/eventsAPI)`.

Reviewer: ```suggestion
  - Provide the Datadog URL and API key.
```

    ![datadog-API-and-URL](./static/datadog-api-conf.png)

  - (Optional) Add headers if needed.

  - Test and complete the setup.

> For details, see Datadog's documentation on [event integrations (v1)](https://docs.datadoghq.com/api/latest/events/) 

Reviewer: ```suggestion
For details, check Datadog's documentation on [Events API (v1)](https://docs.datadoghq.com/api/latest/events/) 
```

</Tabs>

## Datadog notifications

> :::note
> The Datadog notifications feature is currently gated by the feature flag `PIPE_DATADOG_NOTIFICATIONS`, which must be enabled to access this functionality.
> :::

Reviewer 2: ```suggestion
:::note
Currently, the Datadog notifications feature is behind the feature flag `PIPE_DATADOG_NOTIFICATIONS`. Please, contact [Harness Support](mailto:support@harness.io) to enable this feature.
:::
```

The Datadog Notifications in Harness allows you to send pipeline event notifications directly to your Datadog monitoring system using the [Events API (v1)](https://docs.datadoghq.com/api/latest/events/). 

This integration helps you keep track of important pipeline updates within your existing monitoring workflows.

### Prerequisites
  

>   - API Key for your Datadog account: You can create one in Datadog by going to **Profile → Organization Settings → API Keys** or directly via the [API Keys page](https://app.datadoghq.com/organization-settings/api-keys).

Reviewer: Please add DataDog URL as the first prerequisite. Suggestion: Datadog website URL eg. https://app.datadoghq.com

</Tabs>

## Datadog notifications

:::note
The Datadog notifications feature is currently gated by the feature flag `PIPE_DATADOG_NOTIFICATIONS`, which must be enabled to access this functionality.
:::

> The Datadog Notifications in Harness allows you to send pipeline event notifications directly to your Datadog monitoring system using the Events API. 

Reviewer: ```suggestion
The Datadog Notifications in Harness allows you to send pipeline event notifications directly to your Datadog monitoring system using the Datadog Events API (v1) [https://docs.datadoghq.com/api/latest/events/]. 
```

This integration helps you keep track of important pipeline updates within your existing monitoring workflows.

### Prerequisites
  

>   - API Key for your [Datadog account](https://app.datadoghq.com/organization-settings/api-keys). 

Reviewer: We should not add a sub-bullet IMO. We can simply use colon

Reviewer: The link is for API Keys page in Datadog. Should we hyperlink API key instead of Datadog account?

Author: updated

  
    - You can generate one from the API Keys section under `Profile -> Organization Settings -> API Keys` in Datadog.

### Configuration

  - Give a name to your Datadog notification channel.

  - Select Pipeline Events as per your requirements.

  - Select Channel type as `Datadog(/v1/events API)` in Notification Method.

    ![datadog-selection](./static/datadog-notification-1.png)

  - Provide the Datadog URL and API key.

    - Please provide the Datadog URL followed by `/api/v1/events` eg. `https://app.datadoghq.com/api/v1/events/`

      ![datadog-API-and-URL](./static/datadog-api-conf.png)

  - (Optional) Add headers if needed.

  - Test and complete the setup.

> For details, check Datadog's documentation on [Events API (v1)](https://docs.datadoghq.com/api/latest/events/) 

Reviewer: ```suggestion
For details, check Datadog's documentation on [Events API (v1)](https://docs.datadoghq.com/api/latest/events/).
```
